app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ActiveRecord::Base
+  mount_uploader :avatar, AvatarUploader
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  validates :email, presence: true
```

Comment:
Wasn't this on another branch too? Is this branch branched from a branch instead of branched from master?